config/routes.rb

```diff
@@ -73,6 +73,7 @@
       get :print, on: :member
       collection do
         get :pick_ups
+        get ':id/picked_up', to: 'distributions#picked_up', as: :picked_up
```

Comment:
I think this might be better as a filtered parameter rather than a separate route entirely

An exception case would be if this view needed special rendering or preparation -- if it's just pulling a normal collection and listing it, better to use a general collection route instead